package.json

```diff
@@ -5,12 +5,18 @@
   "module": "dist/webexComponents.esm.js",
   "scripts": {
     "build": "rollup -c",
+    "blah": "build-storybook -o ./docs/storybook",
     "prettier": "prettier \"src/**/!(static)/**/*.*(js|jsx)\"",
     "prettier:write": "prettier \"src/**/!(static)/**/*.*(js|jsx)\" --write",
+    "storybook": "start-storybook -p 8080 -h localhost",
+    "storybook:build": "build-storybook -o ./docs/storybook",
+    "storybook:ci": "npm run storybook:clean && npm run storybook:build",
+    "storybook:clean": "rimraf ./docs/storybook",
+    "storybook:watch": "start-storybook -p 8080 -h localhost -w",
```

Comment:
what does this script do that `storybook` doesn't already?
In the current setup `npm start` (now `npm run storybook`) already watches for changes. Also, i don't think that's an [option for `start-storybook`](https://storybook.js.org/docs/configurations/cli-options/#for-start-storybook)

Reply:
Nothing in their docs indicates that `start-storybook` is watching by default

https://storybook.js.org/docs/configurations/cli-options/

Reply:
Not sure why it doesn't say explicitly, but it does. `-w` is not an option fo `start-storybook`

Reply:
Actually, it says it somewhere else, [found it](https://storybook.js.org/docs/guides/guide-react/#finally-run-your-storybook) 😁

Reply:
Actually you're right, i'm mistaken and misread it
Watch is for build-storybook

Reply:
Let's remove this duplicate entry so it doesn't cause confusion.

Reply:
I agree with Adam, `npm run storybook -- -w` should do it.